google/cloud/storage/bucket_metadata.cc

```diff
@@ -42,18 +104,41 @@ BucketMetadata BucketMetadata::ParseFromString(std::string const& payload) {
 bool BucketMetadata::operator==(BucketMetadata const& rhs) const {
   return static_cast<internal::CommonMetadata<BucketMetadata> const&>(*this) ==
              rhs and
+         acl_ == rhs.acl_ and
+         billing_.requester_pays == rhs.billing_.requester_pays and
+         cors_ == rhs.cors_ and default_acl_ == rhs.default_acl_ and
          project_number_ == rhs.project_number_ and
          location_ == rhs.location_ and labels_ == rhs.labels_;
 }
 
 std::ostream& operator<<(std::ostream& os, BucketMetadata const& rhs) {
   // TODO(#536) - convert back to JSON for a nicer format.
-  os << "BucketMetadata={name=" << rhs.name() << ", etag=" << rhs.etag()
-     << ", id=" << rhs.id() << ", kind=" << rhs.kind();
-  char const* sep = "labels.";
+  os << "BucketMetadata={name=" << rhs.name() << ", acl=[";
+  char const* sep = "";
+  for (auto const& acl : rhs.acl()) {
```

Comment:
Nit: Could you add a blank line between each chunk of lines that prints each metadata attribute?  I think that would make this easier to read through for humans :)

Reply:
Done.

Reply:
I don't see any additional whitespace?